spec/models/computacenter/api/cap_usage_update_spec.rb

```diff
@@ -84,5 +84,43 @@
         expect(cap_usage_update.status).to eq('succeeded')
       end
     end
+
+    context 'if the devices_ordered decreases' do
+      let!(:user) { create(:user, :with_a_confirmed_techsource_account, orders_devices: true, responsible_body: school.responsible_body) }
+
+      let(:args) do
+        {
+          'capType' => 'DfE_RemainThresholdQty|Std_Device',
+          'shipTo' => '123456',
+          'capAmount' => 100,
+          'usedCap' => 1,
+        }
+      end
+
+      it 'might send notification to order' do
```

Comment:
can the spec be explicit about the circumstances in which it might or might not send the notification? Just saying it 'might' sounds like it's non-deterministic...